src/main/resources/assets/tconstruct/book/encyclopedia/zh_tw/upgrades/armor/general/tconstruct_freezing.json

```
@@ -0,0 +1,18 @@
+{
+    "modifier_id": "tconstruct:freezing",
+    "text": [
+      {
+        "text": "每級有15%的概率冰凍攻擊者4秒（每級+4秒），施加緩速效果並且造成傷害。多件盔甲的效果將會堆疊。"
```

繁中這邊為 `機率`，其餘地方也需要替換
```suggestion
        "text": "每級有15%的機率冰凍攻擊者4秒（每級+4秒），施加緩速效果並且造成傷害。多件盔甲的效果將會堆疊。"
```